### PR TITLE
FISH-6697 Update Metro to 2.4.8 and Woodstox 6.4.0

### PR DESCRIPTION
**Pull request by arieki**
## Description

Metro upgrade to 2.4.8.payara-p1
Woodstox upgrade to 6.4.0

woodstox upgrade is required to resolve OSGi issue because metro-jax-ws is using 6.x.x version 

### Blockers

https://github.com/payara/patched-src-metro-wsit/pull/17 - metro re-apply patches
https://github.com/payara/patched-src-metro-jax-ws/pull/7 - metro-jax-ws re-apply patches

### New tests

None

### Testing Performed

Build and run payara server
Run payara samples and quicklook test

### Testing Environment

Windows 11, WSL, OpenJDK8, Maven 3.8.6

## Documentation

None

## Notes for Reviewers

None

### CODE REVIEW

File: appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/WSClientContainer.java [hunk 1]
```diff
@@ -60,8 +61,8 @@ public WSClientContainer(ServiceReferenceDescriptor ref) {
 
     public <T> T getSPI(Class<T> spiType) {
         
-        if((spiType == com.sun.xml.ws.assembler.ClientPipelineHook.class) ||
-           (spiType == com.sun.xml.ws.assembler.dev.ClientPipelineHook.class)){
+        if((spiType == com.sun.xml.ws.assembler.metro.dev.ClientPipelineHook.class) ||
+           (spiType == com.sun.xml.ws.assembler.metro.dev.ClientPipelineHook.class)){
```
Comment (svendiedrichsen) [inline appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/WSClientContainer.java:65]: This second one looks redundant.

File: appserver/security/webservices.security/src/main/java/com/sun/enterprise/security/webservices/SecurityServiceImpl.java [hunk 2]
```diff
@@ -37,7 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-// Portions Copyright [2018-2019] [Payara Foundation and/or its affiliates]
+// Portions Copyright [2018, 2019, 2022] [Payara Foundation and/or its affiliates]
```
Comment (Pandrex247) [inline appserver/security/webservices.security/src/main/java/com/sun/enterprise/security/webservices/SecurityServiceImpl.java:40]:
```suggestion
// Portions Copyright [2018-2022] [Payara Foundation and/or its affiliates]
```

File: appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/SecurityService.java [hunk 3]
```diff
@@ -37,7 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-// Portions Copyright [2018] [Payara Foundation and/or its affiliates]
+// Portions Copyright [2018, 2022] [Payara Foundation and/or its affiliates]
```
Comment (Pandrex247) [inline appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/SecurityService.java:40]:
```suggestion
// Portions Copyright [2018-2022] [Payara Foundation and/or its affiliates]
```

File: appserver/security/webservices.security/src/main/java/com/sun/enterprise/security/webservices/ClientPipeCreator.java [hunk 4]
```diff
@@ -37,7 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-// Portions Copyright [2018] [Payara Foundation and/or its affiliates]
+// Portions Copyright [2018, 2022] [Payara Foundation and/or its affiliates]
```
Comment (Pandrex247) [inline appserver/security/webservices.security/src/main/java/com/sun/enterprise/security/webservices/ClientPipeCreator.java:40]: Usual format is with a hyphen, not commas
```suggestion
// Portions Copyright [2018-2022] [Payara Foundation and/or its affiliates]
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
Please update Third Party License.
https://github.com/payara/Payara/blob/master/appserver/packager/legal/src/main/resources/glassfish/legal/3RD-PARTY-LICENSE.txt#L58

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**